src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+#include <boost/algorithm/string.hpp>
+#include <sstream>
 
 namespace ripple {
 
+class Definitions
+{
+private:
+    std::string
+    translate(std::string inp)
+    {
+        auto replace = [&](const char* f, const char* r) -> std::string {
+            std::string out = inp;
+            boost::replace_all(out, f, r);
+            return out;
+        };
+
+        auto find = [&](const char* s) -> bool {
```

Comment:
`find` is already a member function on `string` and it acts differently from this lambda. Consider renaming this. C++ 23 has a `contains` member function that is similar to this. Maybe rename this `contains`?

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/d558b665cd2658b8fd6d2176662758f9efd8ae3f